app/src/main/java/org/p2p/wallet/newsend/ui/linkgeneration/SendLinkGenerationPresenter.kt

```diff
@@ -0,0 +1,36 @@
+package org.p2p.wallet.newsend.ui.linkgeneration
+
+import timber.log.Timber
+import java.math.BigInteger
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.p2p.core.token.Token
+import org.p2p.core.utils.fromLamports
+import org.p2p.wallet.BuildConfig
+import org.p2p.wallet.common.mvp.BasePresenter
+import org.p2p.wallet.newsend.interactor.SendInteractor
+import org.p2p.wallet.newsend.model.LinkGenerationState
+import org.p2p.wallet.newsend.model.TemporaryAccount
+
+class SendLinkGenerationPresenter(
+    private val sendInteractor: SendInteractor
+) : BasePresenter<SendLinkGenerationContract.View>(),
+    SendLinkGenerationContract.Presenter {
+
+    override fun generateLink(recipient: TemporaryAccount, token: Token.Active, lamports: BigInteger) {
+        launch {
+            try {
+                val memo = BuildConfig.sendViaLinkMemo
```

Comment:
memo -> defaultMemo

Reply:
What do you mean by default?
It can be changed, it's not default.
What other memo types do you know?